src/nvim/channel.c

```diff
@@ -322,6 +322,7 @@ static void close_cb(Stream *stream, void *data)
 ///                         stdin disconnected.
 /// @param[in]  cwd  Initial working directory for the job.  Nvim's working
 ///                  directory if `cwd` is NULL
+/// @param[in]  path  The path of the executable. If NULL, use `argv[0]`.
```

Comment:
```suggestion
/// @param[in]  arg0  The path of the executable. If NULL, use `argv[0]`.
```

It's probably helpful if this lives next to `argv`.

Reply:
`arg0` is the name that makes the least sense. I think `exepath` or `cmd` are better.